docs/rholang/rholangtut.md

```diff
@@ -143,6 +143,38 @@ Process-valued variables are bound in process positions in patterns.  Process po
     match R { contract foo(x) = { P } => Q }
     match R { contract foo(@S) = { x!(P + S) } => Q }
 
+In addition to free variables we have the logical connectives "AND", written `/\`, and "OR", written `\/`. In order to match with a list of patterns, separated by `/\`, a process or name must match each pattern in the list. For example, to send a message over `@"chan"` that be received by
+
+    for(@{ @Nil!(x) /\ @y!(Nil) } <- @"chan"){ ... }
```

Comment:
Instead of trying to make the minimal example, try to make the minimal *useful* example.  So for instance, use "RSON" to do something like `for (@{ @"name"!(name) | @"age"!(16 \/ 17) } <- students) { ... }` to select all students that are 16 or 17 years old, or `for (@{ {@"eyes"!("purple")} \/ {@"skin"!("green")} | otherAttributes } <- aliens) { ... }`

Reply:
My students at UVU often criticized me for not giving more relevant examples 😅 Just changed them, hopefully they're more intuitive.